package.json

```diff
@@ -21,11 +21,15 @@
   },
   "homepage": "https://github.com/alleyinteractive/huron-examples#readme",
   "devDependencies": {
-    "css-loader": "^0.28.4",
-    "huron": "file:huron",
-    "node-sass": "^4.5.3",
-    "sass-loader": "^6.0.5",
-    "style-loader": "^0.18.2",
-    "webpack": "^2.6.1"
+    "css-loader": "0.28.4",
+    "node-sass": "4.5.3",
+    "sass-loader": "6.0.5",
+    "style-loader": "0.18.2",
+    "webpack": "2.6.1",
+    "huron": "git+ssh//git@github.com:alleyinteractive/huron.git#master"
```

Comment:
I think we might want to try this with just `alleyinteractive/huron.git#master` as you shouldn't need to authenticate in order to get read access to the repo.